lib/index.test.js

```diff
@@ -49,13 +49,19 @@ describe('pushMetadata', () => {
     const bucket = 'gs://random-bucket';
     await pushMetadata(bucket, data);
 
-    const expectedBucketRegex = new RegExp(/^gs:\/\/random-bucket\/a-service\/github-build\.1234\..{3}\.json$/);
-
     expect(exec.exec.mock.calls.length).toBe(1);
-    expect(exec.exec.mock.calls[0][0]).toBe('gsutil');
-    expect(exec.exec.mock.calls[0][1][0]).toBe('cp');
-    expect(exec.exec.mock.calls[0][1][1]).toBe('./tmp-slipstream-metadata.json');
-    expect(exec.exec.mock.calls[0][1][2]).toMatch(expectedBucketRegex);
+    expect(exec.exec).toHaveBeenCalledWith(
+      'gsutil',
+      expect.arrayContaining([
+        'cp',
+        '-z',
+        'json',
+        expect.stringContaining(
+          'gs://random-bucket/a-service/github-build.1234.'
+        ),
+      ]),
+      {}
+    );
```

Comment:
I was attempted to tweak the existing test, but then my perfectionism kicked in 😉 

https://jestjs.io/docs/en/expect#expectobjectcontainingobject